src/styles/card.css

```diff
@@ -83,12 +82,13 @@
   text-transform: uppercase;
   font-weight: 700;
   font-size: 14px;
-  color: #fff;
+  color: var(--color-white);
 }
 
 .card__button-link:hover {
-  background: #fff;
+  background: var(--color-white);
   border: 1px solid var(--color-blue-accent);
   border-radius: 5px;
   color: var(--color-blue-accent);
+  padding: 11px 69px 11px 68px;
```

Comment:
I have reduced paddings of a button on hover by 1px , as its 1 px border shifts the button's position.
I tried box-sizing: border-box etc., but it does not work (may be because the height / width is not set).
I would appreciate, if you could give a hint how to solve this issue in more sophisticated way.

Reply:
the hint is in the checklist
![image](https://user-images.githubusercontent.com/98764641/212374485-d1f1532f-f9f5-45aa-a489-58a9cc44f839.png)